### PR TITLE
chore(Types): include @discordjs/voice typings

### PR DESCRIPTION
**Pull request by amishshah**
**Please describe the changes this PR makes and why it should be merged:**

- Removed `@discordjs/voice` as a dev dependency
- Includes the adapter typings from `@discordjs/voice`
- Disables `skipLibCheck` to prevent type errors from going unseen

**Status and versioning classification:**

- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating

### CODE REVIEW

**Review by vladfrangu — APPROVED** [review]

**Comment by kyranet** [comment]
I suppose nobody can avoid it, but this PR needs a rebase 😅

**Review by iCrawl — APPROVED** [review]

File: typings/index.d.ts [hunk 1]
```diff
@@ -380,7 +399,7 @@ declare module 'discord.js' {
     public valueOf(): N;
     public [Symbol.iterator](): IterableIterator<S>;
     public static FLAGS: unknown;
-    public static resolve(bit?: BitFieldResolvable<S, N>): number | bigint;
+    public static resolve(bit?: BitFieldResolvable<any, number | bigint>): number | bigint;
```
Comment (iCrawl) [inline typings/index.d.ts:402]: This change was introduced in #5910 but needs to be reverted until a better solution can be found (not in this PR) to fix typings fully before moving forward.

**Review by kyranet — APPROVED** [review]
As long as this is tested with and without `@discordjs/voice`, and the types work fine, LGTM.

**Review by SpaceEEC — APPROVED** [review]
No more `skipLibCheck` 🙏

**Pull request merged**